test/sparse/sparse.jl

```diff
@@ -1843,3 +1843,13 @@ end
     B = A[5:-1:1, 5:-1:1]
     @test issymmetric(B)
 end
+
+@testset "optimizing sparse covariance" begin
+    n = 10000
```

Comment:
Better use a smaller number to limit the memory use when running tests. Should also call `srand(1)` to similar to ensure results are reproducible.